rules/CodingStyle/Application/UseImportsAdder.php

```diff
@@ -253,4 +291,76 @@ private function isCurrentNamespace(string $namespaceName, ObjectType $objectTyp
 
         return ! \str_contains($afterCurrentNamespace, '\\');
     }
+
+    /**
+     * @param Stmt[] $stmts
+     * @param Use_[] $newUses
+     * @return Stmt[]
+     */
+    private function insertUseNodesInStatements(
+        array $stmts,
+        array $newUses,
+        int $position = 0,
+        bool $addSpace = true
+    ): array {
+        $importInsertSorted = SimpleParameterProvider::provideBoolParameter(Option::IMPORT_INSERT_SORTED);
+        if ($importInsertSorted && isset($stmts[$position])
+            && ($stmts[$position] instanceof Use_ || $stmts[$position] instanceof GroupUse)) {
+            foreach ($newUses as $newUse) {
+                do {
+                    $useListPosition = 0;
+                    $prefix = '';
+                    if (! isset($stmts[$position])
+                        || (! $stmts[$position] instanceof Use_ && ! $stmts[$position] instanceof GroupUse)) {
+                        break;
+                    }
+
+                    if ($stmts[$position] instanceof GroupUse) {
```

Comment:
If the current statement is a `GroupUse` statement we need to save the prefix to use it when comparing for sort order